spartan/aztec-network/templates/pxe.yaml

```diff
@@ -101,7 +101,7 @@ spec:
                 - |
                   curl -s -X POST -H 'content-type: application/json' \
                   -d '{"jsonrpc":"2.0","method":"pxe_isGlobalStateSynchronized","params":[],"id":67}' \
-                  127.0.0.1:{{ .Values.pxe.service.nodePort }} | grep -q '"result":true'
+                  127.0.0.1:{{ .Values.pxe.service.nodePort }} | grep -q '"protocolVersion":1'
```

Comment:
?

Reply:
That method doesn't exist anymore, so I replaced it with one that checks PXE is connected to the node. @alexghr WDYT?

Reply:
Hmm, does isGlobalStateSynchronized return protocol version 1?

Reply:
Oh does that come from json rpc? Sounds fine to me